Edits and updates across meetings, onboarding, and culture docs

epbrenner opened the pull request:
Made a few tweaks throughout, including unifying format between sections and adjusting some word choices to fit my persnickety needs. Added information for code of conduct and lab meetings.

epbrenner committed:
Minor edits plus adding conduct and meetings info

Made a few tweaks throughout, including unifying format between sections and adjusting some word choices to fit my persnickety needs. Added information for code of conduct and lab meetings.

jananiravi commented on culture.qmd:
```diff
@@ -13,11 +13,11 @@ If you share our passion, please [reach out to us](//jravilab.github.io/joinus)!
 - serve as a sounding board for all your ideas. You will have the freedom to work on your best ideas broadly aligned with the lab's vision and you will always find in me an audience to discuss and brainstorm.
 - help outline and plan your project(s) and design appropriate analyses.
-- check-in with you once a semester as you plan your time, and more regularly to ensure you're on track -- based on the professional, scientifical, and personal goals you've set for yourself.
+- check-in with you once a semester as you plan your time, and more regularly to ensure you're on track -- based on the professional, scientific, and personal goals you've set for yourself.
```
Good catch! I was deleting from scientifically --> scientific -- stopped midway, I guess. 🤔

epbrenner committed:
Redirected to sections in culture.qmd

Edited to remove repeated info already in culture.qmd, redirecting to appropriate sections where relevant. Minor additions otherwise, e.g. adding Bluesky profile in social media

epbrenner committed:
Adding Mastodon JRaviLab link

Forgot to add lab-specific link for Mastodon under Social media

jananiravi commented on onboarding.qmd:
```diff
@@ -77,6 +77,6 @@ If you haven't already, please create a professional GitHub account (e.g., `jana
 # Social media
 
-If you have officially joined us as a grad or postdoc, or undergrad for 3/6+ months, you will automatically get added to our [group webpage](https://jravilab.github.io/#people). If not, please check a few sample pages and furnish me with those details -- I can add you there right away. Also, we use Twitter for professional networking & announcements (connecting with the broader scientific/R/techn ...
+If you have officially joined us as a grad or postdoc, or undergrad for 3/6+ months, you will automatically get added to our [group webpage](https://jravilab.github.io/#people). If not, please check a few sample pages and furnish me with those details -- I can add you there right away. Also, we use Twitter for professional networking & announcements (connecting with the broader scientific/R/techn ...
```
I am not able to notice the differences. What changed? 🤔

epbrenner replied:
Oh, now I remember -- this is adding the JRaviLab-specific link for Mastodon! Sorry, blanked on what I actually did ten minutes ago.

jananiravi commented on onboarding.qmd:
do we have content for this?

epbrenner replied:
I added some of this info in culture.qmd, but it still needs review.

jananiravi commented on onboarding.qmd:
```diff
@@ -138,6 +77,6 @@ If you haven't already, please create a professional GitHub account (e.g., `jana
 # Social media
 
-If you have officially joined us as a grad or postdoc, or undergrad for 3/6+ months, you will automatically get added to our [group webpage](https://jravilab.github.io/#people). If not, please check a few sample pages and furnish me with those details -- I can add you there right away. Also, we use Twitter for professional networking & announcements (connecting with the broader scientific/R/techn ...
+If you have officially joined us as a grad or postdoc, or undergrad for 3/6+ months, you will automatically get added to our [group webpage](https://jravilab.github.io/#people). If not, please check a few sample pages and furnish me with those details -- I can add you there right away. Also, we use Twitter for professional networking & announcements (connecting with the broader scientific/R/techn ...
```
Did anything change?

epbrenner replied:
This is the one where there is a change, just adding Bluesky. I'm not sure how this became outdated and the duplicate change of seemingly nothing popped up as a second edit.

jananiravi commented on culture.qmd:
```diff
@@ -142,23 +137,39 @@ As a Ph.D. student, you're expected to ...
--   **Collaborate.** We encourage you to present your work and discuss it with others frequently. Learn about the projects your labmates are working on, communicate any troubles or issues before they boil (work-related and not), and keep a positive mental attitude!
--   **Own your research.** This is *your* research process, so be sure that you’re putting forth your best work and that you continue to practice scientific reading and writing.
--   **Take advantage of resources.** There are many avenues for learning in research. Here are a few to get you started:
+-   __Be present.__ Attend meetings and bring forth your best effort.
```
was the bold not rendering?

epbrenner replied:
No, no issues there -- it's just unifying so it's the same format as the italicized points earlier in the same document. I figured it was better if both were either italicized or bolded and went with __the former__ solely because it came first. I can revert if preferred.

jananiravi commented on culture.qmd:
```diff
@@ -171,13 +182,20 @@ Diversity, accessibility
-[More details here]()
-- Research update presentations
-- Giving & receiving constructive feedback
+Lab meetings are divided into a few formats. 
```
we have meetings and schedule in a different doc. Maybe link to that, too?

epbrenner replied:
Oversight on my part. I'll revise and centralize info in meetings.qmd. Thanks!

jananiravi commented on culture.qmd:
did we have this CoC written elsewhere or did you just write it out today?
Also, could we change Ravi Lab --> JRaviLab throughout (for consistency if not anything else)?

epbrenner replied:
This CoC was just a sketch I threw down as a starter. Happy to revise or remove as requested.

Will do a sweep through all docs to update to JRaviLab.

jananiravi commented on culture.qmd:
somewhere here, above, or below, might be worth including a link to this training certificate program. I did the first two modules today and found it very helpful.

https://www.cuanschutz.edu/offices/diversity-equity-inclusion-community/news-events-and-learning-opportunities/heal-trainings-and-workshops/equity-certificate-program

HEAL @ODEICE

jananiravi replied:
Also, we could include them to come to our group, department, OR grad programs to give seminars about any of these topics: https://medschool.cuanschutz.edu/deans-office/diversity-inclusion/training

jananiravi commented on culture.qmd:
Academia has its problems -- generally true, but related to the latter part of the sentence? Should we clarify what we mean by the former?

jananiravi commented on culture.qmd:
```diff
@@ -171,13 +182,20 @@ Diversity, accessibility
-- Giving & receiving constructive feedback
+Lab meetings are divided into a few formats. 
+
+- Weekly lab meetings provide a space for everyone to share updates and roadblocks, and depending on the needs of the group, these meetings can be in a roundtable format where everybody who has updates can share them briefly (<10min per person), longer form updates or presentations for 1-2 people (<30min per person), or the entire slot can be reserved so one person can give a practice talk or maj ...
```
maybe -- convert to short sub-bullets rather than para -- for easy reading? Or if meeting times is elsewhere, we could move this to the other/separate doc? Also, we may have some brief descriptions there. 🤔

epbrenner replied:
Will move to meetings.qmd and link to it instead.

jananiravi commented on culture.qmd:
```diff
@@ -171,13 +182,20 @@ Diversity, accessibility
+
+- Weekly lab meetings provide a space for everyone to share updates and roadblocks, and depending on the needs of the group, these meetings can be in a roundtable format where everybody who has updates can share them briefly (<10min per person), longer form updates or presentations for 1-2 people (<30min per person), or the entire slot can be reserved so one person can give a practice talk or maj ...
+- Weekly project-specific meetings give lab members under a broadly shared project/topic umbrella a space for more detailed discussions and to seek specific feedback and help from their peers in the same or similar topics. These meetings are only required for those in the relevant associated projects, but are open to any members curious to sit in, learn, and  contribute. Like the weekly lab meeti ...
+- One-on-one meetings or small group meetings can be scheduled as-needed, either with Janani or others. 
```
give booking link: bit.ly/janani-cucal

jananiravi commented on culture.qmd:
```diff
@@ -171,13 +182,20 @@ Diversity, accessibility
+- Weekly lab meetings provide a space for everyone to share updates and roadblocks, and depending on the needs of the group, these meetings can be in a roundtable format where everybody who has updates can share them briefly (<10min per person), longer form updates or presentations for 1-2 people (<30min per person), or the entire slot can be reserved so one person can give a practice talk or maj ...
+- Weekly project-specific meetings give lab members under a broadly shared project/topic umbrella a space for more detailed discussions and to seek specific feedback and help from their peers in the same or similar topics. These meetings are only required for those in the relevant associated projects, but are open to any members curious to sit in, learn, and  contribute. Like the weekly lab meeti ...
+- One-on-one meetings or small group meetings can be scheduled as-needed, either with Janani or others. 
+- Journal club or technical primers serve as opportunities to present interesting new research papers from other groups, or to give practice teaching a topic by sharing how-to or tutorial lectures on topics of your choosing. For journal clubs, all members of the lab must read the selected paper ahead of time and be prepared to help their peers explain it. Likewise, to be respectful of the time co ...
```
Up to the trainees to come up with what are appropriate primer topics to give/take as a group and host it with speakers within or outside the group (e.g., DBMI SET, ODEICE, or topic expert from a different lab)

jananiravi commented on culture.qmd:
```diff
@@ -171,13 +182,20 @@ Diversity, accessibility
+- One-on-one meetings or small group meetings can be scheduled as-needed, either with Janani or others. 
+- Journal club or technical primers serve as opportunities to present interesting new research papers from other groups, or to give practice teaching a topic by sharing how-to or tutorial lectures on topics of your choosing. For journal clubs, all members of the lab must read the selected paper ahead of time and be prepared to help their peers explain it. Likewise, to be respectful of the time co ...
+
+Meetings are almost always held in-person, but are held in a hybrid format with Zoom. While it's strongly encouraged for members to attend whenever possible in-person, remote attendance is an option. If you do plan on joining meetings remotely, please communicate this via Slack ahead of time when possible (with our collective understanding that emergencies arise), and you must still make an effor ...
```
remote attendance is an option (esp. if the member has a reasonable request and/or has brought this to the group ahead of time) -- this is to be mindful and respectful of those make the time and effort to show up on time in person.

jananiravi replied:
any brief note on starting on time, having clear agendas and filling out the schedule ahead of time -- so we plan ahead and respect everyone's time?

jananiravi replied:
Naps ✔️ 😆

epbrenner replied:
Good points, will update.

jananiravi commented on culture.qmd:
```diff
@@ -171,13 +182,20 @@ Diversity, accessibility
+In all meetings and interactions, it is imperative that lab members are constructive in feedback and respectful of each other. Critiques are always welcome and encouraged, but they should be supportive and aimed at building others up and improving our science, not tearing others down. At the same time, we shouldn't shy away from providing critical feedback! As scientists, we all need regular prac ...
+
 ### Remote work & working hours
 At a very broad level, the more time you put into your work, the more you’ll get accomplished. However, this is also an obvious oversimplification. Some people prefer a typical 9-5, others thrive with a 10 hour/4 day workweek, and others still find their productivity falls off after 6 hours of work. Adding to this, our work can often be asynchronous, and working remotely can be integrated into a  ...
```
Regardless of your working style, remember to take breaks (e.g., brisk walk, hydration, moderate (!) caffeination). Also, when working remotely for extended periods of time, be sure to hold a few 'co-work times' where you and others can log on and co-work in a shared space allowing spontaneous interactions.

@epbrenner more on 'being present' and spontaneous interactions here or elsewhere in lab engagement/work ethic? Not being present rarely in the group + wearing headphones 100% of the time -- different ways to say, I'm not here to engage with you!

jananiravi reviewed (changes requested):
Requested minor changes. Looks good overall -- thanks for the fixes and additions. I think it reads much better now!

epbrenner committed:
Update culture.qmd per feedback

Made edits per feedback from JR (thanks!), largely removed section on meetings and will update meetings.qmd next to incorporate the removed text where appropriate.

epbrenner committed:
Reorganized meetings.qmd

Merged some changes from culture.qmd and onboarding.qmd into meetings.qmd, restructuring to include a section on what the different meetings are like, and their schedule separately towards the bottom.

epbrenner committed:
Update onboarding.qmd

Added URL to meetings.html instead of culture.html#meetings to use expanded meetings.qmd. Changed a URL to reflect GH repo renaming to "main" without redirect

jananiravi commented on meetings.qmd:
```diff
@@ -4,46 +4,51 @@ Please update this semester's meeting schedule [here](https://docs.google.com/sp
-[Bioinfo primer series](https://github.com/JRaviLab/bioinfo-primers) <br> Thu: 11a--12p MST
+## Logistics
 
-### Monthly Group Socials ☕ 🍵 🥐
```
too bad we don't do these anymore. we should absorb it into the group meeting schedule and include the occasional Lost Coffees!

jananiravi commented on meetings.qmd:
```diff
@@ -4,46 +4,51 @@ Please update this semester's meeting schedule [here](https://docs.google.com/sp
-<TBD>
-
--   Random discussions/catch-up
--   Online games: https://jackbox.tv \| https://drawbattle.io
```
Oh no! all these are gone? Can't we have them in a little section saying if people are up to it, we can do these quasi-regularly?

jananiravi commented on meetings.qmd:
```diff
@@ -52,3 +57,12 @@ Book via [Outlook bookings](https://bit.ly/janani-cucal).
+<TBD>
+
+-   Random discussions/catch-up
+-   Online games: https://jackbox.tv \| https://drawbattle.io
```
good!
